### ISSUE TITLE
about visualize

### ISSUE THREAD

**Issue by Gloria2tt** [HarshayuGirase/PECNet#44]
sorry for bothering 
can you provide the visualize method? I am new to this field , hope you can do me a fever.thank you!

**Comment by tj-cf** [comment]
hello，have you sovled the problem?

**Comment by Gloria2tt** [comment]
> hello，have you sovled the problem?

No, I have changed my research area. Maybe you can try AI tools, they may help you a lot .
